File: UETshare/src/main/java/com/example/uetshare/response/dto/SubCommentDto.java
package com.example.uetshare.response.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.Calendar;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
public class SubCommentDto {

    private Long id;
    private String content;
    private String image;
    private Long comment_id;
    private Calendar time;

}

File: app/src/main/java/ie/app/uetstudents/Entity/subcomment/get/SubcommentDto.kt
package ie.app.uetstudents.Entity.subcomment.get

import ie.app.uetstudents.Entity.subcomment.get.AccountDto

data class SubcommentDto(
    val accountDto: AccountDto,
    val comment_id: Int,
    val content: String,
    val id: Int,
    val image: Any,
    val time: String
)